STAR.Web/Views/Skill/Details.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@using STAR.Web.Models;
+@model SkillModel
+
+<h2>Skills</h2>
+
+@using (Html.BeginForm())
+
+{
+    @Html.AntiForgeryToken()
+    @Html.ValidationSummary(false);
+
+    <div class="form-horizontal">
+            
+        <div class="form-group">
+            @Html.LabelFor(m => m.SkillName)
+            @Html.TextBoxFor(m => m.SkillName, new { @class = "form-control" })
+        </div>
```

Comment:
You can put @Html.ValidateFor(...) so that when ModelState.IsValid is false, the error messages are displayed